plugin-template

```diff
@@ -19,6 +19,7 @@ DEFAULT_SETTINGS = {
     'additional_repos': [],
     'upgrade_range': [],
     'black': True,
+    'flake8': True,
```

Comment:
Any reason why not to expose this in the README and bootstrap code?

Reply:
I just forgot